library/common-test/tests/service/load_balancer_test.yaml

```diff
@@ -0,0 +1,138 @@
+suite: service loadBalancer test
```

Comment:
Figured since we have the integrated loadbalancer to add it, as it wasn't much work, considering most of the it's keys had the templates ready to consume.

Can be removed.

Reply:
is there any reason why rendering tests are removed, but the rendering stays in place?
It means you can potentially run into render mistakes without noticing. We've had that A LOT.

It's absolutely crucial if you want to ever get enterprise customers to ensure 100% (or close to) unittest covered for features.